### ISSUE TITLE
Assistant and likely other buttons have an explicit aria-label attribute breaking any chance of the button having a label on the a11y layer

### ISSUE THREAD

**Issue by tyrylu** [home-assistant/frontend#25370]
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [x] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

The assistant and other buttons as well have an explicit aria-label value set to the empty string. That causes these buttons to have an empty label on the a11y layer as well.

### Describe the behavior you expected

The button labels should be correct also in the a11y tree.

### Steps to reproduce the issue

1. Open the logged in homepage
2. Observe the a11y tree

### What version of Home Assistant Core has the issue?

2025.5

### What was the last working version of Home Assistant Core?

2025.4.3

### In which browser are you experiencing the issue?

Firefox 138, but any browser will be affected, Chromium based browsers somehow get an a11y description, but that only mean that the heuristics differ a little bit

### Which operating system are you using to run this browser?

Linux, but does not matter

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### Javascript errors shown in your browser console/inspector

```txt

```

**Comment by tyrylu** [comment]
This still occurs.

**Comment by tyrylu** [comment]
In 2026.4.4, the situation is a little bit different - there's an aria-labeled-by, but only on the ha-icon-button component, it does not seem to propagate to the button tag which actually gets focus.

**Comment by tyrylu** [comment]
This has been fixed in the 26.x series.
